book/metrics/metrics-ui.md

```diff
@@ -98,31 +98,31 @@ In this section, we'll give a page by page overview of the information presented
 
 ![Application Dashboard Page](/book/metrics/images/application-dashboard-page.png)
 
-Let's look at the elements that make up our Celsius Conversion application. We have a general `Stats Overview Section` for each category.
+Let's look at the elements that make up our Alerts application. We have a general `Stats Overview Section` for each category.
 
-As we can see in our Pipeline Stats section, we're running a single pipeline `Celsius Conversion`. In our Worker Stats section we only have a single worker named `Initializer` processing data for our application. In our Computation Stats section we have 3 computations running: `Add 32`, `Decode Time in TCP Source`, and `Multiply By 1.8`.
+As we can see in our Pipeline Stats section, we're running a single pipeline `Alerts`. In our Worker Stats section we only have a single worker named `Initializer` processing data for our application. In our Computation Stats section we have 1 computation running: `CheckTransaction`.
 
-Let's click into our `Celsius Conversion` pipeline under `Pipeline Stats` to visit its `Detailed Metrics` page.
+Let's click into our `Alerts` pipeline under `Pipeline Stats` to visit its `Detailed Metrics` page.
 
 ### Detailed Metrics
 
 ![Pipeline Detailed Metrics page](/book/metrics/images/pipeline-detailed-metrics-page.png)
 
 #### Stats Overview Section
 
-The `Detailed Metrics` page gives us deeper dive into the stats for a given element within a section. Every `Detailed Metrics` page will include an expanded stats line, also including the `99.9%` and `99.99%` Percentile Bins. Having a look below we can see that `99.99%` of messages processed for the `Celsius Conversion` pipeline are less than or equal to 66 milliseconds. We can also see that we have a median throughput of 49k. Both these stats representing the last 5 minutes of processing data from the incoming stream.
+The `Detailed Metrics` page gives us deeper dive into the stats for a given element within a section. Every `Detailed Metrics` page will include an expanded stats line, also including the `99.9%` and `99.99%` Percentile Bins. Having a look below we can see that `99.99%` of messages processed for the `Alerts` pipeline are less than or equal to 66 milliseconds. We can also see that we have a median throughput of 49k. Both these stats representing the last 5 minutes of processing data from the incoming stream.
```

Comment:
"Both these stats representing" should be maybe "Both of these stats represent" 

Also, nitpicky, but I'd add a comma after: "Having a look below, "